openslides/core/migrations/0002_misc_features.py

```diff
@@ -46,6 +46,21 @@ def name_default_projector(apps, schema_editor):
     Projector.objects.filter(pk=1).update(name='Default projector')
 
 
+def remove_old_countdowns_messages(apps, schema_editor):
+    """
+    Remove old countdowns and messages created by 2.0 from projector elements which are unusable in 2.1.
+    """
+    Projector = apps.get_model('core', 'Projector')
+    projector = Projector.objects.get(pk=1)
+
+    projector_config = projector.config
+    for key, value in list(projector.config.items()):
+        if value.get('name') == 'core/countdown' or 'core/message':
```

Comment:
Are you sure that this condition is interpreted as `value.get('name') == ('core/countdown' or 'core/message)` and not as `(value.get('name') == 'core/countdown') or 'core/message`?

I propose to use: `value.get('name') in ('core/countdown', 'core/message')`.